qcengine/mdi_server.py

```diff
@@ -108,24 +109,34 @@ def __init__(
 
         # Dictionary of all supported MDI commands
         self.commands = {
+            "<@": self.send_node,
             "<NATOMS": self.send_natoms,
             "<COORDS": self.send_coords,
             "<ENERGY": self.send_energy,
             "<FORCES": self.send_forces,
             ">COORDS": self.recv_coords,
             "SCF": self.run_energy,
-            "<NCOMMANDS": self.send_ncommands,
-            "<COMMANDS": self.send_commands,
             "<ELEMENTS": self.send_elements,
             ">ELEMENTS": self.recv_elements,
             "<MASSES": self.send_masses,
+            ">MASSES": self.recv_masses,
             "<TOTCHARGE": self.send_total_charge,
             ">TOTCHARGE": self.recv_total_charge,
             "<ELEC_MULT": self.send_multiplicity,
             ">ELEC_MULT": self.recv_multiplicity,
             "EXIT": self.stop,
         }
 
+        # Register the @DEFAULT node
+        MDI_Register_Node("@DEFAULT")
```

Comment:
I think we need to add a check for the MDI version at the start of the init with these new features.:

```python
from qcelemental.util import parse_version
if parse_version(mdi.__version__) < parse_version("1.0.0"):
    raise ...
```

Reply:
@dgasmith Good catch.  I've added the check.